src/software/js-samples/importUsers.html

```diff
@@ -834,12 +837,16 @@ <h2>Help</h2>
 									user: {"id": userIds[i]}
 								};
 
+								//if no ruleset is selected, do not push one to user
+								if (hosRulesetArray[i] != "") {
```

Comment:
`!=` can be unsafe comparer. JS can do some unexpected things with `!=`. `!==` is safer. Here's some [more info](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons).